silk/tools/otns_manager.py

```diff
@@ -328,6 +332,13 @@ def send_extaddr_event(self):
         self.logger.debug(f"Node {self.node_id} sending extaddr={self.extaddr:016x}")
         self.send_event(event.to_bytes())
 
+    def send_dut_serial_event(self):
+        """Send DUT serial event.
+        """
+        event = Event.status_event(f"dut_serial={self.dut_serial}")
```

Comment:
What is a typical DUT serial look like? 
Do you have an example?

Reply:
It's usually a 9-digits string for nRF boards.

Reply:
Defined in hwconfig.ini. Example:
DutSerial: 683536778